### ISSUE TITLE
Set up initial project environment (backend excluding LLM container)

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#16]
## User story
1. As a team
2. We need to set up the initial project environment with all the services
3. So that all dev team members have a clean environment that includes all DB and services running as one on Docker, excluding the LLM container - separated out to another story #19 .

## Acceptance criteria
* A clean environment where all necessary services and dependencies (excluding LLM) for the backend are created and started using Docker Compose. See image attached
* This will include having a demo of the build process (Homework 3)

![Image](https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/assets/5893774/206d7f90-6354-4e75-a427-6135ab47fb85)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
